cilium-cli/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium \
     make GOARCH=${TARGETARCH} DESTDIR=/out/${TARGETOS}/${TARGETARCH} $(echo $MODIFIERS | tr -d '"') \
     -C cilium-cli install
 
-FROM docker.io/library/ubuntu:24.04@sha256:72297848456d5d37d1262630108ab308d3e9ec7ed1c3286a32fe09856619a782 AS release
+FROM docker.io/library/ubuntu:24.10@sha256:102bc1874fdb136fc2d218473f03cf84135cb7496fefdb9c026c0f553cfe1b6d AS release
```

Comment:
We usually only update within LTS Ubuntu versions.

Reply:
@nebril
Thank you for your comment! Yeah...it should be an LTS release, and 24.10 is not appropriate. I checked the latest [information](https://ubuntu.com/security/CVE-2025-0395) now, and a patch is already available. The issue will be automatically resolved through the [update](https://github.com/cilium/cilium/blob/main/cilium-cli/Dockerfile#L39)?